assignment1.py

```diff
@@ -40,7 +40,10 @@ def plot_unit_sphere(azimuthal_sampler, polar_sampler, num_points):
 
 # Script starts here
 plot_unit_sphere(sample_azimuthal_angle, sample_polar_angle, 2000)
-plot_unit_sphere(sample_azimuthal_angle, sample_incorrect_polar_angle, 2000)
+plot_unit_sphere(sample_azimuthal_angle, sample_incorrect_polar_angle, 2000)   #this should use sample_correct_polar_angle
```

Comment:
Put the comment on another line (preferably the previous line) so that the 80 character limit is enforced.